src/main/java/org/team1540/robot2024/subsystems/drive/ModuleIOSim.java

```diff
@@ -32,7 +31,7 @@ public void updateInputs(ModuleIOInputs inputs) {
         inputs.driveAppliedVolts = driveAppliedVolts;
         inputs.driveCurrentAmps = Math.abs(driveSim.getCurrentDrawAmps());
 
-        inputs.turnAbsolutePosition = new Rotation2d(turnSim.getAngularPositionRad()).plus(turnAbsoluteInitPosition);
+        inputs.turnAbsolutePosition = new Rotation2d(turnSim.getAngularPositionRad());
```

Comment:
should use the static factory methods (fromRadians or fromDegrees) for unit clarity